src/modules/commander/failsafe/framework.cpp

```diff
@@ -204,11 +204,11 @@ void FailsafeBase::notifyUser(uint8_t user_intended_mode, Action action, Action
 			events::send<uint32_t, events::px4::enums::failsafe_action_t, uint16_t, events::px4::enums::failsafe_cause_t>(
 				events::ID("commander_failsafe_enter_hold"),
 			{events::Log::Critical, events::LogInternal::Warning},
-			"{4}, triggering {2} in {3} seconds", mavlink_mode, failsafe_action,
+			"{4}: switching to {2} in {3} seconds", mavlink_mode, failsafe_action,
```

Comment:
Fair enough 👍 
![image](https://github.com/PX4/PX4-Autopilot/assets/4668506/a69e322c-ac16-4766-a207-075906dfd2e7)

Reply:
Why would you change the message to have a colon instead of the comma in some cases? The messages now consistently have a comma to separate issue and reaction. I'd keep this consistency.